### PR TITLE
fix!: Cleanup API definitions, module imports

### PR DESCRIPTION
**Pull request by MasterPtato**
## Changes
- Breaking change caused by updating the route from cloud.games.games to cloud.games in the Fern API

### CODE REVIEW

**Review by AngelOnFira — APPROVED** [review]

File: lib/bolt/core/src/dep/terraform/cli.rs [hunk 1]
```diff
@@ -59,10 +59,7 @@ pub async fn init_if_needed_quiet(ctx: &ProjectContext, plan_id: &str, quiet: bo
 					.read_secret(&["terraform", "pg_backend", "conn_str"])
 					.await
 					.unwrap();
-				vec![
-					format!("-backend-config=conn_str={tf_conn_str}"),
-					"-reconfigure".to_string(),
-				]
+				vec![format!("-backend-config=conn_str={tf_conn_str}")]
```
Comment (NathanFlurry) [inline lib/bolt/core/src/dep/terraform/cli.rs:62]: why was reconfigure removed?

File: lib/util/env/src/lib.rs [hunk 2]
```diff
@@ -194,10 +195,20 @@ pub mod cloudflare {
 		}
 	}
 
+	pub fn account_id() -> &'static str {
```
Comment (NathanFlurry) [inline lib/util/env/src/lib.rs:198]: can't this be simplified to make the lazy_static store the result with EnvVarError instead of an option?

**Review by NathanFlurry — APPROVED** [review]

**Review by NathanFlurry — CHANGES_REQUESTED** [review]
see comments

**Review by NathanFlurry — APPROVED** [review]

**Comment by MasterPtato** [comment]
### Merge activity

* **May 17, 8:01 PM EDT**: [Graphite](https://app.graphite.dev/github/pr/rivet-gg/rivet/534) rebased this pull request after merging its parent, because this pull request is set to merge when ready.

**Pull request merged**